app/Services/StationService.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services;
+
+use App\Station;
+use Illuminate\Database\Eloquent\Collection;
+
+class StationService //Repository?
+{
+
+    public function subscribed(): Collection
+    {
+        //Query getting all stations subscribed by user to current user goes here
+        $stations = Station::all();
+        abort(501);
```

Comment:
Oh, what?

Reply:
Don't mind me testing weird stuff. I was just trying to open discussion about handling the business logic and database-related code (as mentioned on Slack). Let's hope it will get squashed nicely :)